test/test_parmed_formats.py

```diff
@@ -962,6 +964,45 @@ def test_pdb_write_format(self):
         pdbfile.write_pdb(f, write_anisou=True)
         self.assertTrue(diff_files(get_saved_fn('SCM_A_formatted.pdb'), f))
 
+    def test_pdb_write_symmetry_data(self):
+        def assert_remark_290(parm, remark_290_lines):
+            output = StringIO()
+            parm.write_pdb(output)
+            output.seek(0)
+            buffer = output.read()
+            for line in remark_290_lines.split():
+                self.assertTrue(line.strip() in buffer)
+
+        # 4lzt
+        pdbfile = get_fn('4lzt.pdb')
+        parm = pmd.load_file(pdbfile)
+        remark_290_lines = """
+        REMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            
+        REMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            
+        REMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            
```

Comment:
The triple-quotes here mean that the leading whitespace caused by the indentation is part of the lines themselves.  These spaces should be removed (so just remove the indentation from the strings themselves)